src/promptflow-tools/tests/test_aoai.py

```diff
@@ -146,3 +146,97 @@ def mock_completion(self, **kwargs):
             prompt = "dummy_prompt"
             result = completion(connection=conn, prompt=prompt, deployment_name=deployment_name, **params)
             assert result == prompt
+
+    @pytest.mark.skip("Skipping until we have a Azure OpenAI deployment which supports response_format")
```

Comment:
Why need to skip this aoai test? We have '1106' model in CI aoai resource, right?

Reply:
Replace resource https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/96aede12-2f73-41cb-b983-6d11a904839b/resourceGroups/promptflow/providers/Microsoft.CognitiveServices/accounts/promptflow-eus2/overview in CI.